src/foraging_gui/Foraging.py

```diff
@@ -1702,7 +1702,7 @@ def _GetSaveFolder(self,CTrainingFolder=1,CHarpFolder=1,CVideoFolder=1,CPhotomet
         '''The new data storage structure. Each session forms an independent folder. Training data, Harp register events, video data, photometry data and ephys data are in different subfolders'''
         current_time = datetime.now()
         formatted_datetime = current_time.strftime("%Y-%m-%d_%H-%M-%S")
-        self.SessionFolder=os.path.join(self.default_saveFolder, self.Tower.currentText(),self.ID.text(), f'{self.ID.text()}_{formatted_datetime}')
+        self.SessionFolder=os.path.join(self.default_saveFolder, self.ID.text(), f'{self.ID.text()}_{formatted_datetime}')
```

Comment:
I suggest we keep the original folder structure to contain the box so we can easily identify which box the session came from without opening the json file.